src/widgets/conditions/conditions-utils.tsx

```diff
@@ -0,0 +1,14 @@
+import { newWorkspaceItem } from "@openmrs/esm-api";
+
+export const openConditionsWorkspaceTab = (componentToAdd, componentName) => {
+  newWorkspaceItem({
+    component: componentToAdd,
+    name: componentName,
+    props: {
+      match: { params: {} }
+    },
+    inProgress: false,
+    validations: (workspaceTabs: any[]) =>
```

Comment:
Same as above - I would love to see `any[]` being replaced with a more specific type, or otherwise `Array<{ component: string }>`